images/agent/Dockerfile

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+ARG GO_VER
+ARG ALPINE_VER
+
+FROM golang:${GO_VER}-alpine${ALPINE_VER} as golang
```

Comment:
TODO: The image should not require golang. Instead it should be from base alpine.
Build the binary outside and copy into the image.

(the image is now much bigger than it needs to be.)

Reply:
You mean building with `env GOOS=target-OS GOARCH=target-architecture` in host machine. 

Created #606 , will address in upcoming PR

Reply:
@sudeshrshetty we should not build the binary in the host machine

Reply:
we just to make sure that generated image not include golang but the build should still inside in image

Reply:
@sudeshrshetty can you run the generated image and check if golang is exists

Reply:
@troyronda @sudeshrshetty I think the golang is not exist in the generated image we will confirm

Reply:
I double checked generated images, they don't have golang in it. @fqutishat @troyronda